### ISSUE TITLE
[metadata] Decide or develop for spatial resolution

### ISSUE THREAD

**Issue by Ludee** [OpenEnergyPlatform/examples#31]
The spatial resolution is important for the geographic extend.
There is no specification or implementation rule in DCAT-AP that fits. In GeoDCAT-AP is only a possibility as a "comment".

>Spatial resolution (C) 
*Spatial resolution of the dataset (O) 

> rdfs:comment  -  rdfs:Literal 

> See Annex II, Section II.13. Spatial 
resolution is encoded in a human-
readable form only. 

 
Typical spatial resolutions are:
* Political boundaries or [NUTS regions](http://ec.europa.eu/eurostat/web/nuts/background)
* Rasters
* Self-made regions (e.g. eGo Grid Districts)

**Comment by klarareder** [comment]
The spatial resolution usually also contains an extent: e.g. vector or raster grid with a resolution of 0.5 degree (common for global climate models), hence we need a field for the type of spatial resolutions and another field what their extent is - or combine it in one field, but we might get too many differnt options in that case.

**Comment by christian-rli** [comment]
This issue seems outdated @Ludee , can it be closed?

**Comment by Ludee** [comment]
Yes, spatial section has been implemented in the new metadata version.
This issue was concerning the DCAT-AP version of the metadata.
